admin/pt-BR.json

```
@@ -295,7 +295,7 @@
     "SETUP_USER_MESSAGE_2": "Você está tentando acessar o mecanismo de processo \"",
     "SETUP_USER_MESSAGE_3": "\". Esse mecanismo de processo é configurado para usar o serviço de identidade interno (banco de dados), mas não possui usuários administrativos configurados.",
     "SETUP_USER_MESSAGE_4": "Essa página permite que você crie um usuário para acessar o mecanismo de processo.",
-    "SETUP_USER_MESSAGE_5": "Â» Configurar LDAP em vez disso",
+    "SETUP_USER_MESSAGE_5": "Configurar LDAP em vez disso",
```

Hi @professorsoares,

Comparing with the other translations, the "»" character should remain. Please see:

* https://github.com/camunda/camunda-webapp-translations/blob/master/admin/en.json#L299
* https://github.com/camunda/camunda-webapp-translations/blob/master/admin/de.json#L298
* https://github.com/camunda/camunda-webapp-translations/blob/master/admin/es.json#L295

```suggestion
    "SETUP_USER_MESSAGE_5": "» Configurar LDAP em vez disso",
```